webpack/InsightsCloudSync/InsightsCloudSync.js

```diff
@@ -67,13 +67,15 @@ export const generateRuleUrl = ruleId =>
   foremanUrl(`/foreman_rh_cloud/recommendations/${ruleId}`);
 
 const IopRecommendationsPage = props => (
-  <ScalprumComponent
-    scope={scope}
-    module={module}
-    IopRemediationModal={RemediationModal}
-    generateRuleUrl={generateRuleUrl}
-    {...props}
-  />
+  <div className="advisor">
+    <ScalprumComponent
```

Comment:
cc @ColeHiggins2 This may change your locators or whatnot

Reply:
CC @Gauravtalreja1 same